azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -49,7 +54,8 @@
   @Override
   public void uploadExecutableFlow(ExecutableFlow flow)
       throws ExecutorManagerException {
-    flows.put(flow.getExecutionId(), flow);
+    ExecutableFlow exFlow = ExecutableFlow.createExecutableFlowFromObject(flow.toObject());
```

Comment:
It would be great if you can add comments in both places.

My understanding is that the class is simulating a DB layer which should be a copy of the original. Thus cloning is the right choice. Am I right?